### ISSUE TITLE
Player dos not work 1.14.1

### ISSUE THREAD

**Issue by bwl21** [moinejf/abc2svg#29]
When I try

```
X:1
K:C
cdefg
````
on http://moinejf.free.fr/js/edit-1.xhtml it does not play.

**Comment by moinejf** [comment]
It works fine for me. May you give me more information?

**Comment by bwl21** [comment]
I tried once more:
1. goto http://moinejf.free.fr/js/edit-1.xhtml
2. copy the three lines
3. hit play

This morning, I faced an endless loop.
Now I get

![screenshot_928](https://user-images.githubusercontent.com/2022976/30434629-1178bb48-9968-11e7-8081-f2cff1c3809a.jpg)

**Comment by zepedrorocha** [comment]
I was able to reproduce this problem also. But switching the soundfont type from 'mp3' to 'js' it doesn't create this issue

**Comment by moinejf** [comment]
I think I found this problem: the browsers get the mp3 note samples, but they don't give them to the script. It seems this is because cross-site access.
Using 'js' font works because the javascript font is loaded by a <script> tag.

So, I converted the ogg js font to note samples and put them in my site.
Now, the editor of my site should work quicker and without error.

If some editor including toaudio5.js is put either locally in a system or in some other site, playing would work if the note samples are also copied to this system/site.
In either case, using the 'js' soundfont from https://gleitz.github.io/midi-js-soundfonts/FluidR3_GM will continue to work.
